README.md

```diff
@@ -71,7 +71,7 @@ Available targets:
 | github_repositories | List of repository names which should be associated with the webhook | list | `<list>` | no |
 | github_token | GitHub token used for API access | string | - | yes |
 | name | The type of webhook | string | `web` | no |
-| webhook_content_type | Webhook Content Type (E.g. application/json) | string | `application/json` | no |
+| webhook_content_type | Webhook Content Type (E.g. json) | string | `json` | no |
```

Comment:
We need to rebuild the readme using make

make init 
make readme/deps
make readme

Reply:
> We need to rebuild the readme using make
> 
> make init
> make readme/deps
> make readme

@osterman hrmmm I ran ☝️, but something is wrong with the makefile? It overwrote some changes I made to README.md, had to manually fix that too 😅

Reply:
Sorry for the confusion. So our `README.md` is rendered from the `README.yaml`. All terraform input and output descriptions are generated by `terraform-docs`. So if the `README.md` is not correct, then check these other places.